test/database/src/context.cc

```diff
@@ -0,0 +1,31 @@
+#include <gtest/gtest.h>
+
+#include <cstring>
+
+#include <database/context.hh>
```

Comment:
let's use "" instead of <> since this is out 'local' implementation, not external one.